misk/src/main/kotlin/misk/hibernate/Transacter.kt

```diff
@@ -2,4 +2,5 @@ package misk.hibernate
 
 interface Transacter {
   fun <T> transaction(lambda: (session: Session) -> T): T
-}
\ No newline at end of file
+  fun inTransaction(): Boolean
```

Comment:
maybe update the doc of this interface to make it clear `transaction` throws an exception `inTransaction` returns true

Reply:
Updated doc.